src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -25,7 +25,7 @@
 
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 2018,2018 All Rights Reserved
+ * (c) Copyright IBM Corp. 2018,2019 All Rights Reserved
```

Comment:
Pls add the missing space, it should be `2018, 2019`

Reply:
Fixed.